src/shared/bus-util.c

```diff
@@ -1100,14 +1100,17 @@ static int map_basic(sd_bus *bus, const char *member, sd_bus_message *m, sd_bus_
         }
 
         case SD_BUS_TYPE_BOOLEAN: {
-                unsigned b;
-                bool *p = userdata;
+                uint32_t b;
```

Comment:
bools are returned as "int" by sd_bus_message_read_basic(). Hence the above should be an "int", too

Reply:
OK, I will fix so.